### ISSUE TITLE
Difference between previous paper?

### ISSUE THREAD

**Issue by YangJae96** [jiahaoLjh/HumanDepth#3]
Hello. 
Thank you for your great work!!

Compared to the paper in ICCV 2019 Paper, Moon et.al, 
did you only changed the RootNet part to your approach HDNet which estimates the (Xr, Yr,ZR)? 

Is the detection part(Mask-RCNN),  root-relative 3D pose(PoseNet)
and 3D Pose Visualization are all same as the ICCV 2019 paper??

**Comment by jiahaoLjh** [comment]
@YangJae96,

Yes. The bounding box detector and the root-relative 3D pose estimator are the same as in the ICCV paper.

**Comment by YangJae96** [comment]
Thanks!!

Is there a demo code to obtain Roots from my custom images??
![Uploading 캡처.PNG…]()
I want to see the roots like this! Should I change the code?!

**Comment by YangJae96** [comment]
Thanks!!

Is there a demo code to obtain Roots from my custom images??
![root](https://user-images.githubusercontent.com/57695621/91450461-4b683080-e8b7-11ea-8df9-573920547846.PNG)

I want to see the roots like this! Should I change the code?!

**Comment by jiahaoLjh** [comment]
@YangJae96,

For inference on custom images, you will have to edit the data loader accordingly in ```data/dataset.py``` to provide both the image and the bounding box. See #2.

Both the 2D pose and the root joint depth are produced as the output of the model https://github.com/jiahaoLjh/HumanDepth/blob/fba1c6669d09418b1a4bd648a9f4021821ca4037/test.py#L99-L100 which you may consider visualizing with your own code.

**Comment by YangJae96** [comment]
@jiahaoLjh ,

Sorry. I could not understand the reason to fix the dataset.py. 
Isn't dataset.py only for Human36M preprocessing?
If I want to inference my own image(when I have the BBox of human from Detectron2), 
could I just put my image and BBox into the model and get the outputs of 2D joint and root Depth??

![model](https://user-images.githubusercontent.com/57695621/91639337-c749b080-ea50-11ea-85c7-db6a336f335d.PNG)

To test on one single image, which I used Detectron2(Mask-RCNN) to obtain BBox,
I can figure out 
bbox_mask = BBox of one person
vis = number of joints as vector.

**Comment by YangJae96** [comment]
@jiahaoLjh ,

Sorry. I could not understand the reason to fix the dataset.py. 
1. Isn't dataset.py only for Human36M preprocessing?
If I want to inference my own image(when I have the BBox of humans from Detectron2), 
could I just put my image and BBox into the model and get the outputs of 2D joint and root joint Depth??

![model](https://user-images.githubusercontent.com/57695621/91639337-c749b080-ea50-11ea-85c7-db6a336f335d.PNG)
I checked the model input part but its difficult to make in general. 

2. Is the BBox_mask the Bounding box coordinates(x,y) of a one person in an image??

3. I can see the coord_map is for "Normalized image coordinates with focal length 
fx, fy divided from original image coordinates". But if I don't know the focal length of
an image, is there a way to make the input to the model?? 
 
Thanks in advance!!

**Comment by jiahaoLjh** [comment]
@YangJae96 

``dataset.py`` is for preparing input data for the model. You could edit this file to replace data from Human36M with your own image samples. To do that, you need to provide both the image and one bounding box each time (for multi-person case). ``bbox_masks`` is simply a binary mask indicating the region of a bounding box.

If you don't know the focal length, you can simply set a reasonable one by yourself. The ``coord_map`` is taken care by ``dataset.py`` which you don't have to prepare by yourself.
